modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
```

Comment:
Another thing you'll need to do here is bump the version in `package.json` to `5.10.5` since this is the first commit for that.